helm/charts/hydra/values.yaml

```diff
@@ -566,3 +566,8 @@ configmap:
 test:
   # -- Provide additional labels to the test pod
   labels: {}
+
+# -- use a busybox image from another repository
+busybox:
+  repository: busybox
+  tag: 1
```

Comment:
```suggestion
test:
  # -- Provide additional labels to the test pod
  labels: {}
  # -- use a busybox image from another repository
  image:
    repository: busybox
    tag: 1
```

Reply:
Hi there, i think this format would work better, let's keep the test image option in the test object 😉

Reply:
Good idea, the only exception to the rule would be in `helm/charts/oathkeeper/templates/deployment-sidecar.yaml`, where the busybox image is also used. What do you think?

Reply:
Ah, there we use the image as an initcontainer, if you can please make it configurable in the `image` section like 
```yaml
image:
  initContainer:
    repository: foo
    tag: bar
```